src/features/SelectGroup/SelectGroup.scss

```diff
@@ -0,0 +1,14 @@
+div .react-select__control {
```

Comment:
Please create here a class, and use it instead of the divs. It will be a bit safer to it use classes. You can use something similar to `.selectGroup` and put it on the `SelectGroup` component